public/js/components/card-choice.jsx

```diff
@@ -66,22 +66,16 @@ module.exports = React.createClass({
 
   render: function() {
     var cardData = this.props.cards;
-    var cardList = [];
-
     for (var i = 0; i < cardData.length; i += 1) {
       var card = cardData[i];
-      cardList.push(<CardItem {...card} key={'ci-' + i}
```

Comment:
it looks like `key={'ci-' + i}` is lost. Is that ok? Maybe it should move to `CardItem` anyway. If so, I see that the `CardChoice` component is still setting it.